### PR TITLE
Update Web-webapi.http and replaced deprecated DependencyInjection

### PR DESCRIPTION
**Pull request by n0one42**

### CODE REVIEW

**Comment by n0one42** [comment]
- Updated the http file which refers to the wrong parameter (username)
- Replaced the deprecated DependencyInjection with AutoMapper

**Comment by n0one42** [comment]
Duno why this should fail. Also cant see what the error should be. For me it works and I am using it atm exactly like that.

File: src/Application/Application.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Ardalis.GuardClauses" />
-    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" />
+    <PackageVersion Include="AutoMapper" />
```
Comment (jasontaylordev) [inline src/Application/Application.csproj:10]: This should be `<PackageReference Include="AutoMapper" />`. PackageVersion is used in **Directory.Packages.props**.

Comment (n0one42) [reply src/Application/Application.csproj]: Oh... my bad... I overlooked this. Thanks

**Pull request merged**